core/server/master/src/main/java/alluxio/master/file/meta/InodeDirectoryIdGenerator.java

```diff
@@ -36,11 +33,12 @@
  * retrieved.
  */
 @ThreadSafe
-public class InodeDirectoryIdGenerator implements JournalEntryIterable, JournalEntryReplayable {
+public class InodeDirectoryIdGenerator implements Journaled {
+  private static final String NAME = "InodeDirectoryIdGenerator";
```

Comment:
If we change this string, we break the journal?

Reply:
switched to enum to reduce the risk